assets/src/edit-story/components/helpCenter/navigator/components.js

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import styled from 'styled-components';
+/**
+ * Internal dependencies
+ */
+import { Button, themeHelpers } from '../../../../design-system';
+import { NAVIGATION_HEIGHT } from './constants';
+
+export const NavBar = styled.div`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  height: ${NAVIGATION_HEIGHT}px;
+  background-color: ${({ theme }) => theme.colors.bg.secondary};
+  ${themeHelpers.expandTextPreset(({ label }, { MEDIUM }) => label[MEDIUM])}
+`;
+
+export const NavButton = styled(Button)`
+  ${themeHelpers.expandTextPreset(({ label }, { MEDIUM }) => label[MEDIUM])}
```

Comment:
You're kind of the guinea pig for how the text is set up, are you doing this here because you need font size 16px but button size small which would give you a size 14px font?  In cases like that I was kind of envisioning wrapping a `<Text as="span" size="">` inside the `Button` to take priority, but I'm also just curious since this button is breaking with the patterning. It's nothing to change, I just want to understand it in case we need to change how we're going about the text + buttons in the DS or if this is a discrepancy that is just off by a few pixels on accident?

Reply:
yea so I don't think the designs were really using our button here to begin with, so I was trying to make the buttons kind of fit this slightly off use case.

And yea, needed the padding of the small button for things to fit, but the text for these in the designs was larger than in the button size I needed for padding. Basically trying to use our existing components but retaining visual consistency with the designs

I think using a `Text` component or wrapping in a styled component and adding some `themeHelper` styles both work. I don't see a downside to either. They kinda just interface to the same thing so it feels more like developer preference than a need to remove one or the other imo